### DISCUSSION TITLE
Is it possible to disable the bot that asks for the visitor's email?

### DISCUSSION THREAD

**Discussion by ar7hur** [chatwoot/chatwoot#1729] [Q&A]
I've looked through the inbox settings to no avail.

![image](https://user-images.githubusercontent.com/993739/106760522-2df07d00-6634-11eb-8b44-2c1f149801c2.png)

**Comment by sojan-official** [comment]
@ar7hur we have this in our pipeline for the upcoming releases to make this question optional. At the moment it gets triggered for all contacts who don't have an email set.  

one hack would be to use. https://www.chatwoot.com/docs/website-sdk#identity-validation 
and set a random value for the email of the visitor.

**Reply by ar7hur** [reply]
Thanks for your quick reply, and yes I thought about this hack :)
I'll do it until the release.
Cheers!

**Comment by adrianwix** [comment]
What happened with this? My users always get asked for it even if I supplied the email already

<img width="714" alt="image" src="https://github.com/chatwoot/chatwoot/assets/37629556/77b6a2f3-4f33-4b25-9794-10dbba0127cd">

**Comment by youssef1234-code** [comment]
Any Updates on that  ? , same issue after 2 yrs !
